charts/linkerd2/templates/linkerd-config-addons.yaml

```diff
@@ -21,7 +21,7 @@ metadata:
 data:
   values: |-
     global:
-      grafanaUrl:{{.Values.global.grafanaUrl}}
```

Comment:
tioli-- should we quote this so it shows up as `""` when empty?

Reply:
That actually fixes the formatting issue :+1: